pkg/apis/api.acorn.io/v1/types.go

```diff
@@ -332,6 +332,8 @@ type Config struct {
 	PublishBuilders              *bool          `json:"publishBuilders" name:"publish-builders" usage:"Publish the builders through ingress to so build traffic does not traverse the api-server"`
 	BuilderPerProject            *bool          `json:"builderPerProject" name:"builder-per-project" usage:"Create a dedicated builder per project"`
 	InternalRegistryPrefix       *string        `json:"internalRegistryPrefix" name:"internal-registry-prefix" usage:"The image prefix to use when pushing internal images (example ghcr.io/my-org/)"`
+	DefaultWorkloadMemory        int64          `json:"defaultWorkloadMemory" name:"default-workload-memory" quantity:"true" usage:"Set the default memory for acorn workloads." short:"m" default:"256Mi"`
```

Comment:
How about `WorkloadMemoryDefault` to achieve symmetry and help-text nearness (since its sorted alphabetically) to `WorkloadMemoryMaxiumum`?

Reply:
Sounds good to me!